.evergreen/config.yml

```diff
@@ -17,6 +17,12 @@ command_type: system
 # That roughly accounts for variable system performance for various buildvariants
 exec_timeout_secs: 1800 # 6 minutes is the longest we'll ever run
 
+# Ensure that setup and teardown is working as expected.
+pre_error_fails_task: true
+ppre_timeout_secs: 1800 # 5 minutes
+post_error_fails_task: true
+post_timeout_secs: 1800 # 5 minutes
```

Comment:
Confirming, are _all_ the pre and post commands currently defined and used by this EVG config prepared for these fields to be set (failure-resilience)?

Reply:
That is the intent, I'll double-check by scheduling all of the tasks.